clinica/iotools/utils/data_handling.py

```diff
@@ -191,10 +191,10 @@ def create_merge_file(
         if not pipelines:
             for pipeline_name, pipeline_fn in pipeline_options.items():
                 merged_df, summary_df = pipeline_fn(caps_dir, merged_df, **kwargs)
-                if summary_df is not None:
+                if summary_df:
                     merged_summary_df = pd.concat([merged_summary_df, summary_df])
 
-                if summary_df is None or len(summary_df) == 0:
+                if not summary_df or len(summary_df) == 0:
```

Comment:
I wonder whether there is a `len(summary_df) > 0` missing in the first test. Otherwise I don't see why the extra check for `len`is necessary 🤔

Reply:
yes seems like an oversight. Maybe we can add it?